arangod/Agency/AsyncAgencyComm.h

```diff
@@ -168,13 +178,14 @@ class AsyncAgencyComm final {
                                                   velocypack::Buffer<uint8_t>&& body) const;
   [[nodiscard]] FutureResult sendReadTransaction(network::Timeout timeout,
                                                  velocypack::Buffer<uint8_t>&& body) const;
+  [[nodiscard]] FutureResult sendPollTransaction(network::Timeout timeout, uint64_t index) const;
 
+  [[nodiscard]] FutureResult sendTransaction(network::Timeout timeout) const;
```

Comment:
what is that? :D just send anything you like but please respect the timeout!

Reply:
🥇